paymentsheet/src/main/java/com/stripe/android/link/LinkActivityViewModel.kt

```diff
@@ -48,9 +52,11 @@ import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.flow.update
 import kotlinx.coroutines.launch
 import javax.inject.Inject
+import javax.inject.Named
 import com.stripe.android.link.confirmation.Result as LinkConfirmationResult
 
 @SuppressWarnings("TooManyFunctions")
+@NativeLinkScope
 internal class LinkActivityViewModel @Inject constructor(
```

Comment:
👀 Cleaned up DI by deleting `LinkViewModelModule` and putting this `@Inject` annotation to use.